package.json

```diff
@@ -45,7 +46,7 @@
     "elements-core": "yarn workspace @stoplight/elements-core",
     "elements-utils": "yarn workspace @stoplight/elements-utils",
     "elements-dev-portal": "yarn workspace @stoplight/elements-dev-portal",
-    "build": "yarn workspace @stoplight/elements-utils build && yarn workspace @stoplight/elements-core build && yarn workspace @stoplight/elements build && yarn workspace @stoplight/elements-dev-portal build",
+    "build": "yarn workspace @stoplight/elements-utils build && yarn workspace @stoplight/elements-core build && concurrently -n elements-react,dev-portal-react,elements-wc,dev-portal-wc \"yarn workspace @stoplight/elements build.react\" \"yarn workspace @stoplight/elements-dev-portal build.react\" \"yarn workspace @stoplight/elements build.webcomponents\" \"yarn workspace @stoplight/elements-dev-portal build.webcomponents\"",
```

Comment:
I think changing it to from `build` to `build.react` doesn't trigger `build.styles` step making it break in examples as the css file is not compiled and added to dist.

Reply:
Oh, you're right. Thank you!